yellr-serv/yellrserv/ep_client_story.py

```diff
@@ -0,0 +1,59 @@
+from pyramid.view import view_config
+
+import client_utils
+import utils
+
+@view_config(route_name='get_stories.json')
+def get_stories(request):
+
+    result = {'success': False}
+    status_code = 200
+
+    try:
+    #if True:
```

Comment:
are any of these `#if True:` statements worth having? I think if we have exceptions being thrown we should know which blocks are breaking.